File: SMA/SMA.Web/Web/src/app/models/evidencia.ts
import { Reporte } from '../models/reporte';

export class Evidencia {
  public id: number;
  public idReporte: number;
  public nombre: string;
  public extension: string;
  public imagen: string;
  public reporte: Reporte;

  constructor() {
    this.id = 0;
    this.idReporte = 0;
    this.nombre = '';
    this.extension = '';
    this.imagen = '';
  }
}

File: SMA/SMA/Modelos/Evidencia.cs
using System;
using System.Collections.Generic;

namespace SMA.Modelos
{
    public partial class Evidencia
    {
        public int Id { get; set; }
        public int? IdReporte { get; set; }
        public string Nombre { get; set; }
        public string Extension { get; set; }
        public byte[] Imagen { get; set; }

        public virtual Reporte IdReporteNavigation { get; set; }
    }
}
